electrum_nmc/electrum/names.py

```diff
@@ -813,6 +825,21 @@ def get_domain_records_address_freenet(domain, value):
 
     return records, remaining
 
+def get_domain_records_address_dnslink(domain, value):
+    records = []
+    remaining = []
+
+    # Must be string
+    if type(value) != str:
+        return [], value
+
+    if value.startswith("dnslink=/ipfs/"):
+        records.append([domain, "address", ["ipfs", value]])
+    else:
```

Comment:
Please explicitly check for the IPNS prefix here (there's nothing in the dnslink spec that I can find that guarantees these are the only two possible prefixes). If neither the IPFS nor IPNS prefix matches, then I guess `return [], value` unless you think that would break something?

Reply:
Sure, it won't cause any harm, but my rationale was that when we include a dnslink record, we're required to include one of the two options. Therefore, it will consistently be one of those two options.

Reply:
Yes, this will disrupt records that start with `_dnslink.`